test_chime.py

```diff
@@ -35,3 +39,26 @@ def test_script():
 def test_theme_events(theme: str, event: typing.Callable):
     chime.theme(theme)
     assert event(chime) is None
+
+
+@pytest.mark.parametrize('system, expected_config_path',
+                         [('Linux', '/Users/chime/.config/chime/chime.conf'),
+                          ('Darwin', '/Users/chime/.config/chime/chime.conf'),
+                          ('Windows', '/Users/chime/AppData/Roaming/chime/chime.ini')])
+def test__get_config_path(system: str, expected_config_path: str,
+                          monkeypatch: _pytest.monkeypatch.MonkeyPatch):
+    monkeypatch.setenv('HOME', '/Users/chime')
+    monkeypatch.setenv('APPDATA', '/Users/chime/AppData/Roaming')
+    config_path = chime._get_config_path(system)
+    assert config_path.as_posix() == expected_config_path
+
+
+def test__get_default_theme():
+    config_text = textwrap.dedent('''\
+    [chime]
+    theme = zelda
+    ''')
```

Comment:
I would just add a `assert chime.theme() == 'chime'` to make it obvious that the theme has changed :)

Reply:
You pointing this out made me realize there was some fragility around this whole test design 😅, so I ended up refactoring it, but followed your general advice—asserting `chime.theme()` twice to check for a difference.